geonotebook/layers.py

```diff
@@ -150,6 +152,41 @@ def __init__(self, name, remote, vis_url, **kwargs):
         self.vis_url = vis_url
 
 
+class TMSTileLayer(GeonotebookLayer):
+    """Wraps data provided by an external server.
+
+    Note that the data argument is assumed to be of type TMSRasterData.
+    """
+    def __init__(self, name, remote, data, vis_url=None, **kwargs):
+        kwargs['layer_type'] = 'external'
+
+        super(TMSTileLayer, self).__init__(
+            name, remote, data=data, vis_url=vis_url, **kwargs
+        )
+
+        self.data = data
+        self.data.bind("0.0.0.0")
```

Comment:
With the exception of the call to ```bind()``` it seems like ```SimpleLayer``` could be extended to incorporate all this functionality.  This would be desirable,  because then we would not need to check for ```TMSRasterData``` in kernel.py.  

If i am reading this correctly,  the call to ```bind``` is really a feature of the remote TMS Server? In which case i'm not sure the bind function belongs on the TMSRasterData API.  Ideally the data nessisary for bind (e.g. the state maintained on the TMSRasterData object) should be used to call whatever initialization is necessary in the ```ingest``` function. Is this something that could be moved inside the vis_server?